packages/edit-navigation/src/components/layout/index.js

```diff
@@ -43,8 +44,11 @@ const interfaceLabels = {
 	body: __( 'Navigation menu blocks' ),
 	/* translators: accessibility text for the navigation screen settings landmark region. */
 	sidebar: __( 'Navigation settings' ),
+	secondarySidebar: __( 'Block library' ),
 };
 
+const isInserterOpened = true;
```

Comment:
why does this default to true?

Reply:
Oh that's probably an artifact from when I was hacking together the initial implementation. I had to port over a number of variables and I ended up hardcoding them whilst I worked out the correct state, actions and selectors to make the values dynamic.